### ISSUE TITLE
percentage vegetation doesn't sum to 100

### ISSUE THREAD

**Issue by FabianRoger** [FabianRoger/AoB_Butterflies#7]
for about 9285 (38%) of the Site x segment x dates combinations, the perc vegetation does not add up to 100

1) in 132 cases it sums to >100
2) in 7098 cases nothing reported (3 NAs)
3) in 296 cases there is no numeric estimation but an "x" in either category (in all but 1 case only one category is marked)
4) in 1169 cases it sums to less than 100
5) remaining cases include NAs coded as character NA's (the majority), 0 in all categories, and a comment 'Behöver datajustering'

we cannot do much about 2) and 5). 
Not sure how to deal with 1) and 4). 
For 3) we can replace the field marked with an x by 100. 

Note that there are more NAs in cases were the sum is still 100, but there we can assume the NAs code for 0

**Comment by larspett** [comment]
OK. There should only be vegetation information for either transekter (T) or slinga, (S) I think it is transekter, right? Are both S & T included in the sums above?

1.  when the sum is >100, I can check those manually
4. when it sums to less than 100, than could be legit since people could count other surfaces etc so lets just use that
3. yes good choice

For NA at sum=100 then assume 0 as suggested,

**Comment by FabianRoger** [comment]
Yes it includes both. I think Transects should have the information. There is only 64 site x segment x date combination where data are present (as opposed to 6761 were data are present).  For Transect there are only 337 combos for transect with missing data vs 17420 with data. 

If I exclude Segments that reduces the all_NA to 337. 

For the Segments that have data, should I keep it?

**Comment by larspett** [comment]
Hmm I am getting a bit confused with the above. 

So this is Slinga (SiteType=S): There is only 64 site x segment x date combination where data are present (as opposed to 6761 were data are present)?
And this is transect (SiteType=T): For Transect there are only 337 combos for transect with missing data vs 17420 with data?

How do you mean with "If I exclude Segments that reduces the all_NA to 337"

Transects should have the information so whether or not Slingor has it will not matter. But the SLU data (all transects) will be complete will NA etc so the most relevant check will be for the Länsstyrelser (e.g. all exept SLU in the County column).

**Comment by FabianRoger** [comment]
it should have read absent for 6761 for SiteType=S. 

So for SiteType = T, all but 337 SiteID x segment x date combination have data (but some with problems, see above)

For SiteType = S, all but 64 have *no* data. 

The SLU data has some missing data (all NA), the Länstyrelse data has also the X's

**Comment by larspett** [comment]
We have a few cases now that are in email, Two of them are found here and I suggest you patch the raw data. 
For Hommentorp 2013-06-03 seg 6 should be 95/0/5, seg 7 95/0/5, seg 8 30/0/70 and seg 9 55/0/45
For Annehem 2013-06-01, columns should be moved one step left
![image](https://user-images.githubusercontent.com/19598308/104591399-ea29d980-566c-11eb-8520-5267463faea2.png)

**Comment by FabianRoger** [comment]
fixed the cases described above in the raw data. Are there more cases we know how to fix?

**Comment by larspett** [comment]
Bro SV 1 2016-07-18 has also moved one step to the left, should be moved to the right 
![image](https://user-images.githubusercontent.com/19598308/104593626-404c4c00-5670-11eb-85c0-bff313c2cd93.png)

**Comment by larspett** [comment]
Same with Åraslövs mosse 2018-07-06
![image](https://user-images.githubusercontent.com/19598308/104593783-80133380-5670-11eb-802b-00f62b26c0b8.png)

**Comment by larspett** [comment]
![image](https://user-images.githubusercontent.com/19598308/104594358-53abe700-5671-11eb-8992-237957e20215.png)
this should be 30/70

**Comment by larspett** [comment]
Föllingsö 2014-07-20 should be 75/0/25
![image](https://user-images.githubusercontent.com/19598308/104594723-dc2a8780-5671-11eb-9ba4-09d0ddba4b24.png)

**Comment by larspett** [comment]
leave this one NA
![image](https://user-images.githubusercontent.com/19598308/104595169-80143300-5672-11eb-8cb5-c017602e0b8e.png)

**Comment by larspett** [comment]
leave these  two NA
1129 | Djurgården transekter | T | 3 | 2016-06-02

1129 | Djurgården transekter | T | 2 | 2016-06-02

**Comment by larspett** [comment]
OK, apart from those tweaks above, I suggest we set all entries with sums >110 to NA. It looks like 14 cases here. For those with 100 and less, lets assume those are round off errors. OK?

**Comment by FabianRoger** [comment]
Ok, I fixed all the cases above. However I realised that some cases seem to be duplicated with different entries for the same Site x segment x date combination. 

Why would that be? Am I missing a variable for unique ID?

![image](https://user-images.githubusercontent.com/5137092/104608318-05ebaa80-5682-11eb-8349-9faf68b41093.png)
